src/address.js

```diff
@@ -12,7 +12,7 @@ function fromBase58Check (address) {
 
   // TODO: 4.0.0, move to "toOutputScript"
   if (payload.length < 21) throw new TypeError(address + ' is too short')
-  if (payload.length > 21) throw new TypeError(address + ' is too long')
+  if (payload.length > 22) throw new TypeError(address + ' is too long')
```

Comment:
This isn't valid for Bitcoin though.

Perhaps a `network` parameter could be passed to `fromBase58Check` that provides the length paramters?